### PR TITLE
RDoc-2003 applied review comments

### PR DESCRIPTION
**Pull request by reebhub**

### CODE REVIEW

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown [hunk 1]
```diff
@@ -68,8 +68,8 @@ When you connect the database, A list of your database's collection will appear.
 
 !["Retrieved Collection Data"](images/retrieved-collection-data.png "Retrieved Collection Data")
 
-* Your data is loaded, and you can play with it as you will.  
-  One notable field is the rightmost "json()" field; we placed 
+* Your data is loaded, and you can play with it as you wish.  
+  One notable field is the rightmost json() field; we placed 
   it there for irregular data items, should there be ones, that 
   don't fit into one of the otherwise regular json arrays.
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown:74]: 1. the json()  needs to be `json()`

2. Move this bullet to the image below where the user can actually see this column

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown [hunk 2]
```diff
@@ -83,13 +83,14 @@ to import into Power BI just the data you're looking for.
...
 * After providing the URL, port number, and database name,  
-  open **Advanced options**, enter your query into the **SQL Statement** field, 
+  open **Advanced options**, enter your RQL query into the **SQL Statement** field, 
   and click **OK**.  
 
 ---
 
 !["RQL Query Results"](images/rql-query-results.png "RQL Query Results")
 
+* Only the fields resulting from the RQL query will be imported to Power BI.
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown:93]: Add here the bullet about the `json()` field ..

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/settings/integrations.markdown [hunk 3]
```diff
@@ -5,12 +5,15 @@
 
 {NOTE: }
 
+* The Integrations view allows you to define credentials that third-party clients 
+  are required to provide when accessing RavenDB.  
+
 * The **Integrations view** allows you to define credentials, that clients 
   of [software packages bundled with RavenDB](../../../integrations/postgresql-protocol/overview) 
   will be required to provide in order to access RavenDB.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/settings/integrations.markdown:13]: right, removed

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/settings/integrations.markdown [hunk 4]
```diff
@@ -5,12 +5,15 @@
 
 {NOTE: }
 
+* The Integrations view allows you to define credentials that third-party clients 
+  are required to provide when accessing RavenDB.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/settings/integrations.markdown:9]: done

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown [hunk 5]
```diff
@@ -26,27 +26,27 @@
...
 
 ### License
 
-* Your RavenDB license determines which features are active.  
-* Check whether your license activates PostgreSQL, in Studio's **About** Page.  
-  !["About - License"](images/about-license.png "About - License")
-* If your current license doesn't allow you to use PostgreSQL, acquire [one that does](../../studio/server/license-management).  
+* Your RavenDB license determines which features are available.  
+* Visit Studio's **About** page to find which features are included in your license.  
+  !["PostgreSQL and Power BI support is availbale"](images/about-license.png "PostgreSQL and Power BI support is availbale")
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown:37]: done

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown [hunk 6]
```diff
@@ -58,9 +58,9 @@
 
 ### PostgreSQL Port
 
-* To connect RavenDB, your clients need not only its URL but also its PostgreSQL **Port** number.  
+* To connect RavenDB, your clients need not only its **URL** but also its PostgreSQL **Port** number.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown:61]: done

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown:74]: done

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/power-bi.markdown:93]: done

File: Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown [hunk 7]
```diff
@@ -58,9 +60,10 @@
 
 ### PostgreSQL Port
 
-* To connect RavenDB, your clients need not only its URL but also its PostgreSQL **Port** number.  
+* To access RavenDB (or: to connect to RavenDB), your clients need not only
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown:63]: pls choose either option, not both,
I wrote the other option inside the parenthesis - sorry if this was not clear enough

Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/integrations/postgresql-protocol/overview.markdown:63]: right, missed it.
done.

**Review by Danielle9897 — APPROVED** [review]

**Pull request merged**